services/jvm/src/main/scala/io/gearpump/services/MasterService.scala

```diff
@@ -154,6 +154,10 @@ trait MasterService {
                   PartitionerDescription(new PartitionerByClassName(edge))
                 }
 
+                if(graph.hasDuplicatedEdge()){
```

Comment:
This is not right.

1. The frontend will not check this value.
2. new StreamApplication() will throw exception, the route will already return a 500 error.